packages/wdio-browserstack-service/src/cli/modules/BaseModule.ts

```diff
@@ -0,0 +1,56 @@
+import { BStackLogger } from '../cliLogger.js'
+
+/**
+ * Base class for BrowserStack modules
+ */
+export class BaseModule {
+    name: string
```

Comment:
if you provide functions like `getModuleName` to access this property, let's turn these into private fields
```suggestion
    #name: string
```